conferences/abstracts/UKRI-cloud-workshop-abstract.md

```diff
@@ -8,6 +8,6 @@ Sarah Gibson (*The Alan Turing Institute*) on behalf of **The Turing Way Collabo
 
 We aim to support the scientific method across all fields of research using cloud computing platforms to provide reproducible, reliable and reusable analyses. We plan to achieve this through capturing computing environments without need for installation, facilitating access to advanced computing resources and providing a "Safe Haven" for private repositories.
 
-Binder/[BinderHub](https://binderhub.readthedocs.io/en/latest/index.html) is a cloud platform that captures the computing environment of a GitHub repository within a Docker image. It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user. BinderHubs can be built anywhere in the world and hosted in minimal installation time. Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
+[Binder](http://conference.scipy.org/proceedings/scipy2018/project_jupyter.html)/[BinderHub](https://binderhub.readthedocs.io/en/latest/index.html) is a cloud platform that captures the computing environment of a GitHub repository within a Docker image. It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user. BinderHubs can be built anywhere in the world and hosted in minimal installation time. Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
```

Comment:
I think I'd link to the full text PDF rather than just the citation page:

> http://conference.scipy.org/proceedings/scipy2018/pdfs/project_jupyter.pdf

instead of 

> http://conference.scipy.org/proceedings/scipy2018/project_jupyter.html

Also, can you add the citation at the end of the abstract:

> Jupyter, P., Bussonnier, M., Forde, J., Freeman, J., Granger, B., Head, T., … Willing, C. (2018). Binder 2.0 - Reproducible, interactive, sharable environments for science at scale. In Proceedings of the 17th Python in Science Conference. SciPy. https://doi.org/10.25080/majora-4af1f417-011 

It might take you over the word limit so disregard this if that's the case.

Reply:
It leaves me 4 words over, think they'll notice? :joy: I can leave the full reference in here even if it's not directly pasted into the submission form anyway. And I'll include the full reference on the presentation document as well.

Reply:
I don't think they'll notice! And sometimes references don't count anyway 😸 

I just quite like the idea of citing work that doesn't usually get appreciated as much as it should really visibly 👾

Reply:
Yes, definitely. This is now done, any comments on citation style?

Reply:
If you do include the full text for the citation: the first author should be "Project Jupyter" not "Jupyter, P." 😀

Reply:
Thanks!